.github/workflows/ci.yml

```diff
@@ -2,6 +2,7 @@ name: CI
 
 on:
   pull_request:
+    types: [ 'labeled', 'opened', 'synchronize', 'reopened' ]
```

Comment:
Nice idea to use labels to control flow. We could do a `no-netsim` flag to skip netsim for in progress PRs later.